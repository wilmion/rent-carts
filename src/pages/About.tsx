import React from 'react'

import "../sass/pages/about.scss";

const About:React.FC = () => {
    return (
        <section className="about">
            
        </section>
    )
}

export default About
