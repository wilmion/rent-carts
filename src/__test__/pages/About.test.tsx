import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer'; 
import ProviderMock from '../../__mocks__/providerMock';
import About from '../../pages/About';

describe('<About />' , () => {
    test('render' , () => {
        const about = shallow(<About />)
        expect(about.length).toEqual(1)
    })
    test('Snapshot' , () => {
        const about = create(<ProviderMock><About /></ProviderMock>);

        expect(about.toJSON()).toMatchSnapshot();
    })
})