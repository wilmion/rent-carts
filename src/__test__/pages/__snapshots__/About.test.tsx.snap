// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> Snapshot 1`] = `
<section
  className="about"
/>
`;
